tests/test_filtered_wells.py

```diff
@@ -48,22 +51,22 @@ def assert_filtered_inbox_equals_expected(response, expected_data):
     """
 
     assert response.status_code == 200
-    content = FilteredInboxResults.parse_obj(response.json())
-
-    # We try to unpack the data to make expected data again
-
-    actual_data = {}
-
-    for result in content.__root__:
-        wells = {}
-        for well_info in result.wells:
-            wells[well_info.label] = (
-                well_info.qc_status.qc_state
-                if well_info.qc_status is not None
-                else None
-            )
-        actual_data[result.run_name] = wells
-
+    results = response.json()
+    assert type(results) is list
```

Comment:
Is there any need to check data types on a pydantic-constrained return value?

Reply:
The list is explicitly declared as the return value of the end point. We do not have a model for this list.